var/spack/repos/builtin/packages/phasta/package.py

```diff
@@ -0,0 +1,60 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Phasta(CMakePackage):
+    """SCOREC RPI's Parallel Hierarchic Adaptive Stabilized Transient Analysis
+       (PHASTA) of compressible and incompressible Navier Stokes equations."""
+
+    homepage = "https://www.scorec.rpi.edu/software.php"
+    url      = "https://github.com/PHASTA/phasta.git"
+
+    version('master', git='https://github.com/PHASTA/phasta.git',
+        branch='master')
+
+    depends_on('mpi')
+
+    def cmake_args(self):
+        spec = self.spec
+
+        cmake_args = std_cmake_args[:]
```

Comment:
Again, thats from a cut-n-paste and I didn't bother to dig or think more deeply about it ;) I can fix this too.